### ISSUE TITLE
Cannot type in Youtube search

### ISSUE THREAD

**Issue by kaliphx** [Athlon1600/php-proxy-app#147]
Youtube works, but when I try to search something in the search bar of youtube I am unable to type anything. I am able to right click and paste something but the search button also does not work for my self hosted php-proxy.
Does anybody else have a similar problem? could it be an IP related issue?

**Comment by Benji-Collins** [comment]
Try downloading [my fork](https://github.com/Benji-Collins/php-proxy) of php-proxy and hosting that (no need to change files for a quick test). I am able to search fine in mine so if it suddenly works, it was an issue with your installation. If it still doesn't work than there is a larger issue at play.

**Comment by kaliphx** [comment]
I checked your fork, I am not looking to provide HD videos yet since it would effect my content delivery capabilities and most of the user do not have a high speed internet as well.
Moreover I just found out that On a different browser I can type in youtube search through proxy. 
Cannot type in Chrome, Tried Edge, it works fine. not sure if its a extension or what. Although unblockvideos.com youtube search works fine both of the browsers. Just not on my self hosted.
